File: PHP/Miscellaneous/GetReviewsResult.php
<?php
namespace LTD_API;

class GetReviewsResult
{
    //array of Review
    private $reviews;
    private $reviewsCount;
    //instance of double
    private $averageRating;
    private $highestPossibleRating;
    private $currentReviewsCount;
    private $currentReviewsFrom;
    
    public function getReviews()
    {
        return $this->reviews;
    }
    public function setReviews($reviews)
    {
        $this->reviews = $reviews;
    }

    public function getReviewsCount()
    {
        return $this->reviewsCount;
    }
    public function setReviewsCount($reviewsCount)
    {
        $this->reviewsCount = $reviewsCount;
    }

    public function getAverageRating()
    {
        return $this->averageRating;
    }
    public function setAverageRating($averageRating)
    {
        $this->averageRating = $averageRating;
    }

    public function getHighestPossibleRating()
    {
        return $this->highestPossibleRating;
    }
    public function setHighestPossibleRating($highestPossibleRating)
    {
        $this->highestPossibleRating = $highestPossibleRating;
    }

    public function getCurrentReviewsCount()
    {
        return $this->currentReviewsCount;
    }
    public function setCurrentReviewsCount($currentReviewsCount)
    {
        $this->currentReviewsCount = $currentReviewsCount;
    }

    public function getCurrentReviewsFrom()
    {
        return $this->currentReviewsFrom;
    }
    public function setCurrentReviewsFrom($currentReviewsFrom)
    {
        $this->currentReviewsFrom = $currentReviewsFrom;
    }
}

File: Java/Miscellaneous/GetReviewsResult.java
package LTD_API;

import java.util.List;

public class GetReviewsResult
{
    private List<Review> reviews;
    private int reviewsCount;
    private double averageRating;
    private int highestPossibleRating;
    private int currentReviewsCount;
    private int currentReviewsFrom;
    
    public List<Review> getReviews()
    {
        return reviews;
    }
    public void setReviews(List<Review> reviews)
    {
        this.reviews = reviews;
    }

    public int getReviewsCount()
    {
        return reviewsCount;
    }
    public void setReviewsCount(int reviewsCount)
    {
        this.reviewsCount = reviewsCount;
    }

    public double getAverageRating()
    {
        return averageRating;
    }
    public void setAverageRating(double averageRating)
    {
        this.averageRating = averageRating;
    }

    public int getHighestPossibleRating()
    {
        return highestPossibleRating;
    }
    public void setHighestPossibleRating(int highestPossibleRating)
    {
        this.highestPossibleRating = highestPossibleRating;
    }

    public int getCurrentReviewsCount()
    {
        return currentReviewsCount;
    }
    public void setCurrentReviewsCount(int currentReviewsCount)
    {
        this.currentReviewsCount = currentReviewsCount;
    }

    public int getCurrentReviewsFrom()
    {
        return currentReviewsFrom;
    }
    public void setCurrentReviewsFrom(int currentReviewsFrom)
    {
        this.currentReviewsFrom = currentReviewsFrom;
    }
}

